recipes-bsp/u-boot/files/secure-boot.cfg

```diff
@@ -1,5 +1,5 @@
 ### Secure boot config
-# CONFIG_DISTRO_DEFAULTS is not set
+CONFIG_NETDEVICES=y
```

Comment:
Why this one is required?

Reply:
We need that config enabled so that the MAC addresses are forwarded to the DTB. But maybe it stays on, now that DISTRO_DEFAULTS are left in place. Will check again.

Reply:
We get this without it:
```
  LD      u-boot
aarch64-linux-gnu-ld.bfd: board/siemens/iot2050/board.o: in function `set_board_info_env':
/data/u-boot/build-iot2050/../board/siemens/iot2050/board.c:213: undefined reference to `eth_env_set_enetaddr_by_index'
```
Reason remains the same: We use this to transfer the MAC address from the EEPROM to the DTB.